Directory.build.targets

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
   <PropertyGroup Condition="$(TargetFramework.StartsWith('uap'))">
     <DefineConstants>$(DefineConstants);NETFX_CORE;XAML;WINDOWS;WINDOWS_UWP;UWP</DefineConstants>
-    <TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16299.0</TargetPlatformMinVersion>
```

Comment:
Why can't we support 10.0.16299? Xamarin.Forms still support 10.0.16299 and 10.0.14393

Reply:
Because Forms advices to support 17763, and will break in the future.